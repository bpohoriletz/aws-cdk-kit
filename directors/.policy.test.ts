import PolicyDirector from './policy';
import PolicyBuilder from '../iam/policy-builder';

describe('new PolicyDirector()', () => {
  let instance: PolicyDirector;

  beforeEach(() => {
    instance = new PolicyDirector(PolicyBuilder);
  });

  test('creates empty policy', () => {
    expect(instance).toMatchSnapshot();
  });
});
