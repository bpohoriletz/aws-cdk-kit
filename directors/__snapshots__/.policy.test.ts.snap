// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`new PolicyDirector() creates empty policy 1`] = `
PolicyDirector {
  "builder": PolicyBuilder {},
}
`;
